plantlist.rb

```diff
@@ -0,0 +1,15 @@
+class PlantList
+  def initialize(data)
+		@name = data[:name]
+		@description = data[:description]
+    @image =  data[:thumbnail_url]
```

Comment:
when doing "https://www.growstuff.org/crops.json", is the data for each crop different from what is returned for the calls we were doing until now? If not, I was wondering if it would help to return an array of plant objects rendered as json so that our users can potentially add them to their garden if they wish

Reply:
yes it is different from what the calls are doing now. scaled-down data for the most part. The way I am setting it up is to send an array to the view that can be iterated over. What would be the reason for sending an array from the microservice to the main app? Also, I don't know how to do this so I would need you to jump in and help.

Reply:
I was hoping we'd be getting enough data back to create a list of plants objects that could be rendered as json. Would have given users the option to add plants from this list to their garden. But with scaled down data, what you're sending is what we need